src/api/route-services/__mocks__/gem2s.js

```diff
@@ -0,0 +1,16 @@
+const mockGem2sCreate = jest.fn(() => Promise.resolve({
+  stateMachineArn: 'arn:aws:states:eu-east-1:3139055674710:stateMachine:biomage-gem2s-production-2367819df529419d7c95167336f5d6ae681859b2',
+  executionArn: 'arn:aws:states:eu-east-1:3139055674710:execution:biomage-gem2s-production-2367819df529419d7c95167336f5d6ae681859b2:90b4bc06-507e-41e3-ae0b-fc9cc83cnm3f',
```

Comment:
in the mocks, better use the name which we use to call loca/testing environment instead of `production`. Can you change `production` to what we use?

Reply:
change to `development`

Reply:
changed to `testing`